Makefile

```diff
@@ -1,88 +1,93 @@
-########################################################################
-# Development
 
-tools-install:
-	@./scripts/tools-install.sh
+########################################################################
+## Development
 
-tools-versions:
-	@./scripts/tools-versions.sh
+compress:
```

Comment:
When I simply run `make` on my system, it complains about `upx` missing. I think make runs the first target by default. We should better move the `build` target at the top. Simply running `make` should run `build` I think.

Reply:
Fixed with commit 15c69bf4567bb0da8d41a51bcdfb49434df1ac06